addons/explosives/functions/fnc_canDaisyChainTo.sqf

```diff
@@ -0,0 +1,31 @@
+/*
+ * Author: Loren Luke Leimer
+ * Checks if a unit can daisy chain to an explosive
+ *
+ * Arguments:
+ * 0: Unit <OBJECT>
+ * 1: Target Explosive <OBJECT>
+
+ *
+ * Return Value:
+ * Able to Daisy Chain to<BOOL>
+ *
+ * Example:
+ * if ([player,_explosive] call ACE_Explosives_fnc_canDaisyChain) then {hint "Can Daisy Chain";};
+ *
+ * Public: Yes
+ */
+#include "script_component.hpp"
+private ["_explosive","_target","_explosive","_result"];
+_unit = _this select 0;
```

Comment:
Can use `PARAMS(_unit,_target)` here and then you won't need to define `_unit` or `_target` above.